runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/control/FnApiControlClientTest.java

```diff
@@ -100,7 +99,8 @@ public void testRequestError() throws Exception {
   }
 
   @Test
-  public void testUnknownResponseIgnored() throws Exception {
+  public void testUnknownResponse() throws Exception {
+    thrown.expect(NullPointerException.class);
```

Comment:
My opinion: an NPE always indicates a programming error. Null should not exist anymore in high-level programming. Or in most medium-level or low-level programming. The code should prevent an NPE and give a good error message.

Reply:
I agree that meaningful message is better than NPE.

Reply:
I agree that meaningful message is better than NPE.